### PR TITLE
Add instructions for Notion import tool

### PR DESCRIPTION
**Pull request by akhismat**
* Add readme with instractions and examples for Notion import tool
* Add required env variable to run-local command

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NmUwOGZiNTU3NmVmNDVmNWU2ZWY3NGEiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIn0.aHq8paggpL_l0Sho1dwe0wZY4m5E7aax6LCPCRRFlEE">Huly&reg;: <b>UBERF-8065</b></a></sub>

### CODE REVIEW

File: dev/tool/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+--workspace ws1 \
+--teamspace imported
+```
+
+#### For clouds:
+##### Here is an example for those who's using huly.app cloud:
+To import Notion workspace with teamspaces
+```
+docker run \
+  -e SERVER_SECRET="" \
```
Comment (haiodo) [inline dev/tool/README.md]: May be we fix tool to not ask for secret if not used?
ACCOUNTS_URL could be retrieved from FRONT_URL/config.json so better to add this fix as well.

File: dev/tool/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+### Examples
+
+#### For local run:
+When importing Notion workspace with teamspaces
+```
+rushx run-local import-notion-with-teamspaces /home/Export-fad9ecb4-a1a5-4623-920d-df32dd423743 \
+--user john.doe@gmail.com \  
+--password qwe1234 \       
+--workspace ws1
```
Comment (haiodo) [inline dev/tool/README.md]: is ws1 is workspaceUrl? It would be good if it is url.

Comment (akhismat) [reply dev/tool/README.md]: done

File: dev/tool/README.md (hunk 1 above)
Comment (akhismat) [reply dev/tool/README.md]: done

**Comment by akhismat** [comment]
Comments are adressed in https://github.com/hcengineering/platform/pull/6719

**Pull request closed without merging**